recipes/scikit-build-core/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "scikit-build-core" %}
+{% set version = "0.1.0b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
+  sha256: 3d6bcc8ec84972a928962dd8894b82e24390f461a26fede87b4678dd1e98a8d6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - hatch-vcs
+    - hatchling
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - cmake
+    - distlib
+    - exceptiongroup
+    - importlib-resources
+    - make
+    - ninja
+    - packaging
+    - pathspec
+    - pyproject-metadata
+    - tomli
+    - typing-extensions >=3.7
+
+test:
+  imports:
+    - scikit_build_core
+  commands:
+    - pip check
+    - pytest -vv -m "not isolated" -k "not sdist_time_hash_set_epoch and not sdist_hash"
+  source_files:
+    - tests/
+    - pyproject.toml
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pip
+    - build
+    - cattrs >=22.2
+    - pybind11
+    - pytest >=7.2
+    - pytest-subprocess
+    - rich
+    - setuptools
+    - wheel
+
+about:
+  home: https://pypi.org/project/scikit-build-core/
+  summary: Build backend for CMake based projects
+  license: Apache-2.0
+  license_file:
+    - LICENSE
+    - src/scikit_build_core/resources/find_python/Copyright.txt
+
+extra:
+  recipe-maintainers:
+    - conda-forge/scikit-build
+    - henryiii
```

```suggestion
    - henryiii
    - jcfr
    - thewtex
```

Adding @thewtex and myself (@jcfr) to the list of maintainer.

---

@thewtex Are you happy to be added as a maintainer here?

---

Yes, thanks!